cmd/collectiondir/serve.go

```diff
@@ -455,6 +456,26 @@ func getLimit(c echo.Context, min, defaultLim, max int) int {
 	return lv
 }
 
+type xrpcError struct {
+	Error   string `json:"error"`
+	Message string `json:"message"`
+}
+
+func (srv *collectionServer) errorHandler(err error, c echo.Context) {
+	code := http.StatusInternalServerError
+	var errorMessage string
+	if he, ok := err.(*echo.HTTPError); ok {
+		code = he.Code
+		errorMessage = fmt.Sprintf("%s", he.Message)
+	}
+	if code >= 500 && code <= 599 {
+		slog.With("err", err).Error("collectiondir-http-internal-error")
```

Comment:
Is there an advantage to the `With()` syntax, versus:

```
slog.Error("collectiondir-http-internal-error", "err", err)
```

I don't think it matters much either way, just curious